application/jaxrs-redis-quarkus/src/main/java/webapp/tier/service/RedisService.java

```diff
@@ -142,6 +142,10 @@ public MsgBean publishMsg(Jedis jedis) throws RuntimeException, NoSuchAlgorithmE
 
 	@Override
 	public void run() {
+		subscribeRedis();
+	}
+
+	void subscribeRedis() {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5948487)